anndata/_core/aligned_mapping.py

```diff
@@ -1,26 +1,34 @@
+from __future__ import annotations
+
+import warnings
 from abc import ABC, abstractmethod
 from collections import abc as cabc
+from collections.abc import Iterator, Mapping, Sequence
 from copy import copy
-from typing import Union, Optional, Type, ClassVar, TypeVar  # Special types
-from typing import Iterator, Mapping, Sequence  # ABCs
-from typing import Tuple, List, Dict  # Generic base types
-import warnings
+from typing import (  # Special types  # ABCs  # Generic base types
```

Comment:
```suggestion
from typing import (
```

Reply:
those are the changes that should only be done before we merge it because they only become relevant after autofixing

Reply:
or maybe not, we can fix this in the commit before. hm.